src/commonHelper.ts

```diff
@@ -0,0 +1,62 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+'use strict';
+
+import * as vscode from 'vscode';
+import * as fs from 'fs';
+import * as strings from './strings';
+import * as Constants from './constants';
+import { CommandObserver } from './CommandObserver';
+import * as compareVersions from 'compare-versions';
+const regex = /(?<="Microsoft.DataTools.Schema.Tasks.PostgreSql.Sdk\/)(.*?)(?=")/;
```

Comment:
1) Define constant where it's needed, rather than globally?
2) Why do we need look behind? Why not just match it and have a named group for the version portion?
3) Instead of ```.*?```, can't we match ```[^"]+```